tests/tools/test_logs.py

```diff
@@ -1,641 +1,450 @@
-            "",
-        )
-
-        result = await mcp.call_tool("get_journal_logs", {})
-
-        assert isinstance(result, tuple)
-        assert len(result) == 2
-        assert isinstance(result[0], list)
-        output = result[0][0].text
-
-        assert "=== Journal Logs (last 100 entries, no filters) ===" in output
-        assert "Started some service" in output
-        assert "Stopped some service" in output
-
-        # Verify journalctl was called with correct arguments
-        mock_execute_command.assert_called_once()
-        args = mock_execute_command.call_args[0][0]
-        assert args[0] == "journalctl"
-        assert "-n" in args
-        assert "100" in args
-        assert "--no-pager" in args
-
-    async def test_get_journal_logs_with_unit_filter(self, mocker):
-        """Test get_journal_logs with unit filter."""
-        mock_execute_command = mocker.patch("linux_mcp_server.tools.logs.execute_command")
-        mock_execute_command.return_value = (
-            0,
-            "Jan 01 12:00:00 host nginx[123]: Started nginx.",
-            "",
-        )
-
-        result = await mcp.call_tool("get_journal_logs", {"unit": "nginx.service"})
-
-        assert isinstance(result, tuple)
-        assert len(result) == 2
-        assert isinstance(result[0], list)
-        output = result[0][0].text
+from contextlib import nullcontext as does_not_raise
 
-        assert "unit=nginx.service" in output
-        assert "Started nginx" in output
+import pytest
 
-        # Verify unit filter was passed
-        args = mock_execute_command.call_args[0][0]
-        assert "-u" in args
-        assert "nginx.service" in args
-
-    async def test_get_journal_logs_with_priority_filter(self, mocker):
-        """Test get_journal_logs with priority filter."""
-        mock_execute_command = mocker.patch("linux_mcp_server.tools.logs.execute_command")
-        mock_execute_command.return_value = (
-            0,
-            "Jan 01 12:00:00 host systemd[1]: Error occurred.",
-            "",
-        )
+from linux_mcp_server.server import mcp
 
-        result = await mcp.call_tool("get_journal_logs", {"priority": "err"})
 
-        assert isinstance(result, tuple)
-        assert len(result) == 2
-        assert isinstance(result[0], list)
-        output = result[0][0].text
+async def assert_tool_result_structure(result):
+    """Verify common result structure returned by all tools."""
+    assert isinstance(result, tuple)
+    assert len(result) == 2
+    assert isinstance(result[0], list)
+    return result[0][0].text
 
-        assert "priority=err" in output
-        assert "Error occurred" in output
 
-        # Verify priority filter was passed
-        args = mock_execute_command.call_args[0][0]
-        assert "-p" in args
-        assert "err" in args
+@pytest.fixture
+def mock_path_exists(mocker):
```

Comment:
Sorry, I don't think that this fixture provides much value to the test module...